test/images/dnsutils/Dockerfile

```diff
@@ -1,3 +1,17 @@
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
shouldn't this be using the standard boilerplate (Kubernetes authors) instead of Google?